.gitignore

```diff
@@ -30,3 +30,6 @@ adobe-paas.code-workspace
 
 template.
 tmp/
+
+# dropin registry (auto-generated)
+dropin-registry.json
```

Comment:
What is this file, where does it come from, and why can't it be committed?

Reply:
Good catch. That was an earlier file I used for removing dropins. Removed.